<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NuevoMovimiento</class>
 <widget class="QDialog" name="NuevoMovimiento">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>304</width>
    <height>415</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Nuevo Movimiento</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>350</y>
     <width>181</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>166</width>
     <height>29</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Categorias</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="comboCateg"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget1">
   <property name="geometry">
    <rect>
     <x>31</x>
     <y>21</y>
     <width>149</width>
     <height>29</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Cuentas</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="comboCuentas"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>157</width>
     <height>24</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QRadioButton" name="radioGasto">
      <property name="text">
       <string>gasto</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioIngreso">
      <property name="text">
       <string>ingreso</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget3">
   <property name="geometry">
    <rect>
     <x>31</x>
     <y>131</y>
     <width>161</width>
     <height>29</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Importe</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineImporte"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>170</y>
     <width>234</width>
     <height>29</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Descripcion</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineDes"/>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>321</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>Opcional</string>
   </property>
   <widget class="QWidget" name="layoutWidget5">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>230</width>
      <height>29</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Periocidad:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboPeriocidad">
       <item>
        <property name="text">
         <string/>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Cada mes</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Cada dos meses</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Cada año</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>103</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Programación: </string>
    </property>
   </widget>
   <widget class="QDateTimeEdit" name="dateTimeEdit">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>30</y>
      <width>131</width>
      <height>27</height>
     </rect>
    </property>
    <property name="dateTime">
     <datetime>
      <hour>0</hour>
      <minute>0</minute>
      <second>0</second>
      <year>2015</year>
      <month>1</month>
      <day>1</day>
     </datetime>
    </property>
    <property name="date">
     <date>
      <year>2015</year>
      <month>1</month>
      <day>1</day>
     </date>
    </property>
    <property name="maximumDate">
     <date>
      <year>3000</year>
      <month>12</month>
      <day>28</day>
     </date>
    </property>
    <property name="minimumDate">
     <date>
      <year>1788</year>
      <month>9</month>
      <day>14</day>
     </date>
    </property>
    <property name="currentSection">
     <enum>QDateTimeEdit::DaySection</enum>
    </property>
    <property name="displayFormat">
     <string>dd/MM/yyyy</string>
    </property>
    <property name="calendarPopup">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <zorder>groupBox</zorder>
  <zorder>buttonBox</zorder>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>NuevoMovimiento</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>NuevoMovimiento</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
